bionic/flow.py

```diff
@@ -1231,9 +1232,34 @@ def get(self, name, collection=None, fmt=None, mode=object):
             return set(values)
         elif collection is pd.Series or collection == "series":
             if len(result_group.key_space) > 0:
+                case_keys = [result.query.case_key for result in result_group]
+
+                resultgroups_by_entity_name = {
```

Comment:
nit: `result_groups_by_entity_name`

Also, I would recommend prefixing all the ancestor-related values with something `ancestor_` -- that would make it clearer what's going on in a line like `case_key.project(result_case_key.space)`, where it's not obvious that we're projecting an *original* case key onto the key space of an *ancestor*.